src/bin/bindgen.rs

```diff
@@ -217,6 +217,16 @@ pub fn main() {
 
     let mut bind_args: Vec<_> = env::args().collect();
 
+    let clang_version = clang_version();
+
+    if cfg!(feature = "llvm_stable") {
+        assert_eq!(clang_version, "3.8.0");
+    } else {
+        assert_eq!(clang_version, "3.9.0");
```

Comment:
It seemed like it was still up in the air whether or not we wanted to try and parse the version string -- for example, this wouldn't handle a `3.X.1` release.